migrations/20230503114054-update-location-migration.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    /**
+     * Updates the migrated Locations to only set the name and address when structured address was missing
+     * and also set address as locationName if missing (enabling slightly better migration to autocomplete input)
+     */
+    await queryInterface.sequelize.query(`
+      UPDATE "Locations" l
+      SET 
+        "name" = CASE
+                  WHEN l."structured" IS NULL THEN COALESCE(c."locationName", c."address")
+                  ELSE NULL
+                END,
+        "address" = CASE
+                      WHEN l."structured" IS NULL THEN c."address"
+                      ELSE NULL
+                    END
```

Comment:
Won't this drop all existing names and addresses for locations that don't have a `structured` set?

Reply:
It should not, it's only when `structured` exists that we set the `name` and `address` to null.

Reply:
It should not, it's only when `structured` exists that we set the `name` and `address` to null.